tests/numerics/project_bv_curl_conf_complex.output

```diff
@@ -0,0 +1,423 @@
+    61 = 0
+    62 = (0.500000,0.00000)
+    63 = 0
+    66 = (1.50000,0.00000)
+    67 = 0
+    68 = (1.50000,0.00000)
+    69 = 0
+    70 = 0
+    71 = 0
+    72 = 0
+    73 = 0
+    74 = 0
+    75 = 0
+    76 = 0
+    77 = 0
+    82 = 0
+    83 = 0
+    84 = 0
+    85 = 0
+    96 = (1.00000,0.00000)
+    97 = 0
+    98 = (1.00000,0.00000)
+    99 = 0
+    100 = (0.500000,0.00000)
+    101 = 0
+    102 = (1.00000,0.00000)
+    103 = 0
+    106 = (0.500000,0.00000)
+    107 = 0
+    108 = (1.50000,0.00000)
+    109 = 0
+    110 = (1.50000,0.00000)
+    111 = 0
+    112 = 0
+    113 = 0
+    114 = 0
+    115 = 0
+    120 = 0
+    121 = 0
+    122 = 0
+    123 = 0
+    124 = 0
+    125 = 0
+    126 = 0
+    127 = 0
+    138 = (1.00000,0.00000)
+    139 = 0
+    140 = (0.500000,0.00000)
+    141 = 0
+    142 = (1.00000,0.00000)
+    143 = 0
+    144 = (0.500000,0.00000)
+    145 = 0
+    146 = (1.50000,0.00000)
+    147 = 0
+    148 = 0
+    149 = 0
+    150 = 0
+    151 = 0
+    152 = 0
+    153 = 0
+    154 = 0
+    155 = 0
+    156 = 0
+    157 = 0
+    158 = 0
+    159 = 0
+    170 = (1.00000,0.00000)
+    171 = 0
+    172 = (1.00000,0.00000)
+    173 = 0
+    174 = (0.500000,0.00000)
+    175 = 0
+    176 = (0.500000,0.00000)
+    177 = 0
+    178 = (1.50000,0.00000)
+    179 = 0
+    180 = (1.50000,0.00000)
+    181 = 0
+    182 = (1.50000,0.00000)
+    183 = 0
+    186 = 0
+    187 = 0
+    188 = 0
+    189 = 0
+    194 = 0
+    195 = 0
+    196 = 0
+    197 = 0
+    202 = 0
+    203 = 0
+    204 = 0
+    205 = 0
+    212 = (1.00000,0.00000)
+    213 = 0
+    214 = (0.500000,0.00000)
+    215 = 0
+    216 = (0.500000,0.00000)
+    217 = 0
+    218 = (1.50000,0.00000)
+    219 = 0
+    220 = (1.50000,0.00000)
+    221 = 0
+    222 = 0
+    223 = 0
+    224 = 0
+    225 = 0
+    226 = 0
+    227 = 0
+    228 = 0
+    229 = 0
+    234 = 0
+    235 = 0
+    236 = 0
+    237 = 0
+    244 = (1.00000,0.00000)
+    245 = 0
+    246 = (1.00000,0.00000)
+    247 = 0
+    248 = (0.500000,0.00000)
+    249 = 0
+    250 = (1.50000,0.00000)
+    251 = 0
+    252 = (1.50000,0.00000)
+    253 = 0
+    254 = 0
+    255 = 0
+    256 = 0
+    257 = 0
+    262 = 0
+    263 = 0
+    264 = 0
+    265 = 0
+    266 = 0
+    267 = 0
+    268 = 0
+    269 = 0
+    276 = (1.00000,0.00000)
+    277 = 0
+    278 = (0.500000,0.00000)
+    279 = 0
+    280 = (1.50000,0.00000)
+    281 = 0
+    282 = 0
+    283 = 0
+    284 = 0
+    285 = 0
+    286 = 0
+    287 = 0
+    288 = 0
+    289 = 0
+    290 = 0
+    291 = 0
+    292 = 0
+    293 = 0
+DEAL::Tests with FE_NEdelecSZ:
+DEAL::dim=2:
+    0 = (-1.00000,0.00000)
+    1 = 0
+    4 = (-0.500000,0.00000)
+    5 = 0
+    12 = (-1.00000,0.00000)
+    13 = 0
+    14 = (-0.500000,0.00000)
+    15 = 0
+    22 = (-1.00000,0.00000)
+    23 = 0
+    26 = (-0.500000,0.00000)
+    27 = 0
+    32 = (-1.00000,0.00000)
+    33 = 0
+    34 = (-0.500000,0.00000)
+    35 = 0
+DEAL::dim=3:
+    0 = (-1.00000,0.00000)
+    1 = 0
```

Comment:
The result for NedelecSZ is different. Is this the expected behavior?

Reply:
Does this also happen for Number=double and FESystem with two FE_Nedelec/FE_NedelecSZ?

Reply:
Yes, the same result are obtained for `Number=double`.
If I run the test `numerics/project_bv_curl_conf` that is already in the repository, all the results are multiplied by -1 when I use `FE_NedelecSZ`

Results of `numerics/project_bv_curl_conf` for `FE_Nedelec`:
```
DEAL::dim=3:
    0 = 1.00000
    1 = 0
    2 = 1.00000
    3 = 0
    4 = 0.500000
    5 = 0
    6 = 0.500000
    7 = 0
```
Results of `numerics/project_bv_curl_conf` for `FE_NedelecSZ`:
```
DEAL::dim=3:
    0 = -1.00000
    1 = 0
    2 = -1.00000
    3 = 0
    4 = -0.500000
    5 = 0
    6 = -0.500000
    7 = 0
```

Reply:
As @masterleinad  mentioned in https://github.com/dealii/dealii/issues/8323#issuecomment-504011421 In case the shape functions just differ by a sign, this should still be fine.

Reply:
Yes. Individual constraints may have opposite signs. You should look at the *values of the solution at individual points*, rather than the *values of the degrees of freedom*.